pkgs/by-name/bo/bombsquad/package.nix

```diff
@@ -0,0 +1,95 @@
+{
+  lib,
+  stdenv,
+  targetPlatform,
+  fetchurl,
+  python312,
+  SDL2,
+  libvorbis,
+  openal,
+  libgcc,
+  commandLineArgs ? "",
+  makeWrapper,
+  makeDesktopItem,
+  autoPatchelfHook,
+}:
+let
+  archive =
+    {
+      x86_64-linux = {
+        name = "BombSquad_Linux_x86_64";
+        sha1 = "047df21e523e4929e89a5e1836b3b8a57f7779ba";
+      };
+      aarch-64-linux = {
+        name = "BombSquad_Linux_Arm64";
+        sha1 = "7ba2266da8482b4609f3e1f7f37aa12de58fcf66";
+      };
+    }
+    .${targetPlatform.system} or (throw "${targetPlatform.system} is unsupported.");
+in
+stdenv.mkDerivation (finalAttrs: {
+  name = "bombsquad";
+  version = "1.7.34";
+  sourceRoot = ".";
+  src = fetchurl {
+    url = "https://files.ballistica.net/bombsquad/builds/${archive.name}_${finalAttrs.version}.tar.gz";
+    sha1 = archive.sha1;
```

Comment:
Prefer `hash = ...` to `sha1 = ...` Both currently work, but the former will make it easier if the algorithm ever needs to change.

Since there could be different hashing algorithms in use, this make sense.

More info: https://nixos.org/manual/nixpkgs/stable/#fetchurl

> Nixpkgs contributors are currently recommended to use hash.

Reply:
Oh yes, that makes sense. I've used `nix hash file ~/Downloads/BombSquad_Linux_Arm64_1.7.34.tar.gz` to compute the hashes and put them in the derivation.